src/oss-find-squats-lib/Mutators/AfterSeparatorMutator.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.CST.OpenSource.FindSquats.Mutators
+{
+    /// <summary>
+    /// Generates mutations for if a character was changed after a separator was used.
```

Comment:
nit: I believe I left it as a generic comment on @jpinz's PR, but in the future it would be nice to have samples of the mutator in its class doc comment

Reply:
Definitely worth doing a future doc/examples pass on all the mutators.

Reply:
https://github.com/microsoft/OSSGadget/issues/284